packages/installations/src/helpers/get-installation-entry.test.ts

```diff
@@ -354,12 +354,19 @@ describe('getInstallationEntry', () => {
       const { registrationPromise } = await installationEntryPromise;
 
       // Let the new getInstallationEntry process start.
-      await sleep(10);
+      await sleep(50);
 
-      expect(await get(appConfig)).to.deep.equal({
+      const tokenDetails = (await get(
+        appConfig
```

Comment:
Is there a workaround for this formatting? :P

Reply:
I could add a `prettier-ignore` comment but I'd rather not.